code/datums/brain_damage/phobia.dm

```diff
@@ -42,52 +54,137 @@
+		stress --
+		stress_check = world.time + min(3000, (600 * stress))
+	switch(fearscore) //updating the fear state is handled in mob_life, as well as effects like adrenaline rush, handled when the state changes. other things are handled lower
+		if(-INFINITY to 2) //there is a bit of a grace period before you begin to get scared
+			if(fear_state > PHOBIA_STATE_CALM)
+				fear_state = PHOBIA_STATE_CALM
+				to_chat(owner, "<span class ='notice'>You calm down completely.</span>")
+		if(3 to 8)
+			if(fear_state >= PHOBIA_STATE_UNEASY)
+				fear_state = PHOBIA_STATE_EDGY
+				owner.remove_movespeed_modifier(MOVESPEED_ID_PHOBIA, TRUE)
+				to_chat(owner, "<span class ='notice'>you manage to calm down a little</span>")
+			if(fear_state == PHOBIA_STATE_CALM)
+				fear_state = PHOBIA_STATE_EDGY
+				if(prob(stress * 10))
+					fearscore = 9
+		if(9 to 16)
+			if(fear_state >= PHOBIA_STATE_FIGHTORFLIGHT)
+				fear_state = PHOBIA_STATE_UNEASY
+				to_chat(owner, "<span class ='notice'>You're safe now... better be careful anyways.</span>")	
+				owner.add_movespeed_modifier(MOVESPEED_ID_PHOBIA, TRUE, 100, override=TRUE, multiplicative_slowdown = 1)	
+				psychotic_brawling.remove(owner)
+			if(fear_state <= PHOBIA_STATE_EDGY)
+				fear_state = PHOBIA_STATE_UNEASY
+				owner.add_movespeed_modifier(MOVESPEED_ID_PHOBIA, TRUE, 100, override=TRUE, multiplicative_slowdown = 1)	
+				owner.Jitter(5)
+				if(prob(stress * 10))
+					fearscore = 17
+		if(17 to 36)
+			if(fear_state >= PHOBIA_STATE_TERROR) //we don't get an adrenaline rush when calming down
+				fear_state = PHOBIA_STATE_FIGHTORFLIGHT 
+				to_chat(owner, "<span class ='notice'>It's gone for now... Better get out of here before it comes back.</span>")
+				owner.add_movespeed_modifier(MOVESPEED_ID_PHOBIA, TRUE, 100, override=TRUE, multiplicative_slowdown = -0.4)	
+			if(fear_state <= PHOBIA_STATE_UNEASY) //ADRENALINE RUSH! You get psychotic brawling, a burst of speed, and some stun avoidance for awhile. If you fail to escape or destroy the threat during an adrenaline rush, you're fucked either way
+				fear_state = PHOBIA_STATE_FIGHTORFLIGHT
+				to_chat(owner, "<span class ='userdanger'>YOU HAVE TO GET OUT OF HERE! IT'S DANGEROUS!</span>")
+				owner.add_movespeed_modifier(MOVESPEED_ID_PHOBIA, TRUE, 100, override=TRUE, multiplicative_slowdown = -0.4)//while terrified, get a speed boost
+				owner.emote("scream")
+				if(prob(stress * 10))
+					fearscore = 27 //we don't get the adrenaline rush, and keel over like a baby immediately
+				psychotic_brawling = new(null)
+				psychotic_brawling.teach(owner, TRUE)
+				owner.adjustStaminaLoss(-75)
+				owner.SetStun(0)
+				owner.SetKnockdown(0)
+				owner.SetImmobilized(0)
+				owner.SetParalyzed(0)
+				if(owner.handcuffed)
+					owner.visible_message("<span class ='danger'>[owner] starts frantically wrestling with their restraints!.</span>", "<span class ='danger'>I'm trapped! I gotta get out, NOW!.</span>")
+					stoplag(80)
+					owner.uncuff()
```

Comment:
I don't really like the idea of someone just magically uncuffing themselves because they saw something scary after 8 seconds no matter what.

[owner] starts frantically wrestling with their restraints!.</span>
Billy club goes brrrr

Reply:
Also double punctuation